server/pbench/lib/pbench/__init__.py

```diff
@@ -146,40 +221,44 @@ def actions_tracking_closure(cl_actions):
             es, generator, raise_on_error=False,
             raise_on_exception=False, request_timeout=_request_timeout)
 
-    for ok, resp in streaming_bulk_generator:
+    for ok, resp_payload in streaming_bulk_generator:
         retry_count, action = actions_deque.popleft()
-        assert action['_id'] == resp[_op_type]['_id']
+        try:
+            resp = resp_payload[_op_type]
+            status = resp['status']
+        except KeyError as e:
+            assert not ok
+            # resp is not of expected form
+            print(resp)
```

Comment:
Log this somewhere instead of printing it?

Reply:
Yes, this where I was thinking of changing to use a python logger, but felt that we should get this deployed for now and deal with that later.